docs/docstring_template.md

```diff
@@ -0,0 +1,72 @@
+# Docstring Template
+--------------------
+
+Below is a docstring example. It contains a suggested docstring layout, example text and example MathJax.
+
+The layout below will render properly in Read the Docs. Please adjust the example text and MathJax as required.
+
+Note: backticks render differently in docstrings to elsewhere. Should you wish to include text in backticks, please use double backticks - e.g. ``scores.continuous.mean_error``. (If you only use one set of backticks, the text will instead be rendered as italics in Read the Docs). 
+
+```
+def function_name(*args, *, **kwargs) -> type hint:
+    '''
+    A short one or two line description of what the function does. 
+
+    Additional information (e.g. a paragraph or two) may be included here following the short description.
+
+    MathJax giving the mathematical formula for the metric. Below is some MathJax to use as a 
+    starting point. Note that double-backslash formatting is required to ensure proper rendering in docstrings.
+
+    .. math::
+        \\text{mean error} =\\frac{1}{N}\\sum_{i=1}^{N}(x_i - y_i)
+        \\text{where } x = \\text{the forecast, and } y = \\text{the observation}	
```

Comment:
This needs to be split into two sections, otherwise the second line becomes part of the first equation. I suggest just using :math:` around the x and y